src/flutter/shell/platform/linux_embedded/surface/native_window_drm.cc

```diff
@@ -27,6 +27,12 @@ NativeWindowDrm::NativeWindowDrm(const char* deviceFilename) {
     LINUXES_LOG(ERROR) << "Couldn't open " << deviceFilename;
     return;
   }
+
+  if (!drmIsMaster(drm_device_)) {
+    LINUXES_LOG(ERROR) << "Failed to become a DRM master";
```

Comment:
The embedder couldn't become the DRM master. Please confirm if another display backend such as X11 and Wayland is not running.